postgresql-s3-backup/values.yaml

```diff
@@ -0,0 +1,50 @@
+# Default values for progresql-s3-backup.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+cronjob:
+  # At 01:00 every day
+  schedule: "*/5 * * * *"
+  annotations: {}
+  concurrencyPolicy: ""
+  failedJobsHistoryLimit: ""
+  successfulJobsHistoryLimit: ""
+
+config: 
+  s3:
+    region: 'eu-west-2'
+    access_key_id: 'AKIAISGOFSURAVAOOBSA'
+    secret_access_key: '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'
+    bucket: 'stackworx-k8-dev-database-backup/backup'
+    prefix: 'backup'
+  postgres:
+    database: ''
+    user: 'postgres'
+    password: 'pYFgF8uLFz' 
```

Comment:
You should never check access keys into github. these are public. Remind me to revoke those credentials when I get a chance. Granted I know I rushed you a bit to commit this

Reply:
Revoked